scripts/sct_check_dependencies.py

```diff
@@ -154,15 +159,6 @@ def main():
     # check RAM
     sct.checkRAM(os_running, 0)
 
-    path_sct = os.environ.get("SCT_DIR", os.path.dirname(os.path.dirname(__file__)))
-    print('SCT path: {0}'.format(path_sct))
-
-    # fetch SCT version
-    install_type, sct_commit, sct_branch, version_sct = sct._git_info()
-    print('Installation type: %s' % install_type)
-    print('  version: ' + version_sct)
-    print('  commit: ' + sct_commit)
```

Comment:
i guess this is displayed for all SCT commands (including `sct_check_dependencies`) so we can indeed get rid of it